src/palace.pyx

```diff
@@ -246,7 +248,7 @@ def use_fileio(factory: Optional[Callable[[str], FileIO]],
         alure.FileIOFactory.set(unique_ptr[alure.FileIOFactory](
             new CppFileIOFactory(fileio_factory, buffer_size)))
 
-
+
```

Comment:
What happened here?

Reply:
the form feed?

Reply:
What did you do to it?

Reply:
*insert an form feed

Reply:
It's not a formfeed.  Please be serious and keep the part out of the scope of this patch untouched.

Reply:
It's a formfeed. Have a look on group chat that I can actually send some images